src/modules/fundingManager/oracle/interfaces/IFM_PC_ExternalPrice_Redeeming_v1.sol

```diff
@@ -0,0 +1,185 @@
+
+    /// @notice	Emitted when a new redemption order is created.
+    /// @param	orderId_ Order identifier.
+    /// @param	seller_ Address selling tokens.
+    /// @param	receiver_ Address who receives the redeemed tokens.
+    /// @param	sellAmount_ Amount of tokens to sell.
+    /// @param	exchangeRate_ Current exchange rate.
+    /// @param	collateralAmount_ Amount of collateral.
+    /// @param	feePercentage_ Fee percentage applied.
+    /// @param	feeAmount_ Fee amount calculated.
+    /// @param	redemptionAmount_ Final redemption amount.
+    /// @param	collateralToken_ Address of collateral token.
+    /// @param	redemptionTime_ Time of redemption.
+    /// @param	state_ Initial state of the order.
+    event RedemptionOrderCreated(
+        uint indexed orderId_,
+        address indexed seller_,
+        address indexed receiver_,
+        uint sellAmount_,
+        uint exchangeRate_,
+        uint collateralAmount_,
+        uint feePercentage_,
+        uint feeAmount_,
+        uint redemptionAmount_,
+        address collateralToken_,
+        uint redemptionTime_,
+        RedemptionState state_
+    );
+
+    // -------------------------------------------------------------------------
+    // View Functions
+
+    /// @notice	Gets the current open collateral redemption amount.
+    /// @return	amount_ The total amount of open redemptions.
+    function getOpenRedemptionAmount() external view returns (uint amount_);
+
+    /// @notice	Gets the next available order ID.
+    /// @return	orderId_ The next order ID.
+    function getNextOrderId() external view returns (uint orderId_);
+
+    /// @notice	Gets the current order ID.
+    /// @return	orderId_ The current order ID.
+    function getOrderId() external view returns (uint orderId_);
+
+    /// @notice Gets the project treasury address.
+    /// @return address_ The address of the project treasury.
+    function getProjectTreasury() external view returns (address);
+
+    /// @notice Gets the direct operations only flag.
+    /// @return Whether only direct operations are allowed.
+    function getIsDirectOperationsOnly() external view returns (bool);
+
+    /// @notice Gets current buy fee.
+    /// @return buyFee The current buy fee.
+    function getBuyFee() external view returns (uint);
+
+    /// @notice Gets the maximum fee that can be charged for buy operations.
+    /// @return maxBuyFee_ The maximum buy fee.
+    function getMaxBuyFee() external view returns (uint maxBuyFee_);
+
+    /// @notice Gets the maximum project fee that can be charged for sell 
+    ///         operations.
+    /// @return maxProjectSellFee_ The maximum project sell fee percentage.
+    function getMaxProjectSellFee()
+        external
+        view
+        returns (uint maxProjectSellFee_);
+
+    /// @notice Gets current sell fee.
+    /// @return sellFee The current sell fee.
+    function getSellFee() external view returns (uint);
```

Comment:
Same as in the contract, unnamed return value.

Reply:
added